### PR TITLE
Testing tool

### PR DESCRIPTION
**Pull request by jonty-esterhuizen**

### CODE REVIEW

**Review by johanfol — APPROVED** [review]

File: testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/Readme.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+[15:32] Jonty Esterhuizen
```
Comment (joeygoksu) [inline testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/Readme.md:1]: We can remove this line

File: testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/Readme.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+# TMS-Service
+
+The TMS service is a NodeJs REST-API service that ingests a FRM transaction and validates if it has the correct format.
```
Comment (joeygoksu) [inline testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/Readme.md:5]: It's not a REST service. It's a gRPC service that using a single REST endpoint to be activated. (or better explanation)

File: testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/hello-world.spec.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import { credentials } from 'grpc';
+import { CustomerServiceClient, CustomerRequestId, Empty } from './src/grpc/helloworld';
+import chalk from "chalk";
+describe('hello world client checks via grpc', () => {
+    let client: CustomerServiceClient;
+    it('should check that jest is running', (done) => {
+        expect(1).toEqual(1);
```
Comment (joeygoksu) [inline testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/hello-world.spec.ts:7]: Do we really need to check 1=1? If Jest is working the rest of the test should be run.

File: testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/hello-world.spec.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        done();
+    });
+    beforeEach(() => {
+        // create grpc client to send data 
+        client = new
+            CustomerServiceClient('127.0.0.1:30043', credentials.createInsecure());
```
Comment (joeygoksu) [inline testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/hello-world.spec.ts:13]: We can add port and URL in Jest environment variables.

File: testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/hello-world.spec.ts [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    it('should create the client', (done) => {
+        console.log(chalk.green('client connecting to =>'),
+            chalk.blueBright(client.getChannel().getTarget()
+            ));
+        expect(client).toBeDefined();
+        done();
+    })
+    it('should test client status ', (done) => {
+        client.getStatus(new Empty(), (callback, response) => {
+            expect(callback).toBeNull();
```
Comment (joeygoksu) [inline testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/hello-world.spec.ts:25]: I'm not sure this test is working.

**Review by joeygoksu — CHANGES_REQUESTED** [review]

**Comment by joeygoksu** [comment]
We're moved to the new [organization](https://github.com/actiofrm), there is no need to keep this PR open.
@jonty-esterhuizen informed ✅

**Pull request closed without merging**